web/client/components/misc/ComboboxWithEnhancer.jsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+
+const React = require('react');
+// const Combobox = require('react-widgets').Combobox;
+const PagedCombobox = require('./PagedCombobox');
+
+const {addStateHandlers} = require('./enhancer/basic');
+
+// component enhanced with props from stream, and local state
+const ComboboxEnhanced =
```

Comment:
Is it an Enhancer or Enhanced? But first of all ask yourself what is the purpose of this component and then give it a name that is consistent with its purpose.

Reply:
i'll remove ComboboxEnhanced  and put the callback inside the enhancer.

ComboboxWithEnhancer.jsx is a comoponent enhanced/with enhancer.. can you suggest a better name?

Reply:
No, because I don't understand what it can be used for, but maybe you do